packages/verification-vue-component/src/style/components/verification/_item_list.scss

```diff
@@ -105,9 +106,12 @@
                 height: 100%;
 
                 > .header {
-                    height: 65px;
+                    height: 78px;
                     padding: 20px;
                     background-color: $white;
+                    background-repeat: no-repeat;
+                    background-position: 12px 5px;
+                    background-image: var(--cf-verification-logo-url);
```

Comment:
The background size should be defined, probably the height fix and width auto to make sure the passed logo always fits.